examples/rn-connection-and-error/frontend/app/providers/StoreProvider.tsx

```diff
@@ -59,9 +58,12 @@ const StoreContext = createContext<StoreContextType>({
  */
 export function StoreProvider({children}: PropsWithChildren) {
   const realm = useRealm();
-  const store = useQuery(Store)[0];
+  const user = useUser<{}, {storeId: BSON.ObjectId}, {}>();
```

Comment:
We need to improve these generic type signatures 😿

Reply:
We need to improve many things regarding types.  I didn't want to poke that dragon in this PR 🐉